zebrunner.sh

```diff
@@ -75,6 +75,11 @@
     down
   }
 
+  version() {
+      source .env
+      echo "${TAG_JENKINS_MASTER}"
```

Comment:
add info about module name in echoing
echo "jenkins-master: ${TAG_JENKINS_MASTER}"